cmd/kubeadm/app/cmd/platform/init_windows.go

```diff
@@ -0,0 +1,99 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package platform
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"os/user"
+	"strings"
+)
+
+// InitCopyCredentialsForUser will copy the cluster admin credentials to the home path of a non-root user
+func InitCopyCredentialsForUser(copyCredentialsForUser string, adminKubeConfigPath string) error {
```

Comment:
Why can you use ENV-var's or some other platform agnostic library to determine $HOMEDIR.  

/cc @kubernetes/sig-windows-misc

Reply:
this targets any Windows user. the env. variable `%HOME%` can only be used for the current user. also some user might have decided to move his home dir outside of default user path on Windows  - e.g. `/users/`.

cc @luxas 
some related changes of mine in the got accepted in the `user` package of the Go standard libarary...
https://github.com/golang/go/commits?author=neolit123

Reply:
^ these are going to make it in go 1.11 in August i would guess.

Reply:
I'm generally not ok with this change todo the last step which folks automate to add this platform specific code which can/will be obviated.

Reply:
even with my contribs to the `user` package in the go std lib that can eventually replace the above code, this change would still need the `_unix`, `_windows` files, because:
- there is no `chown` on windows
- the way to check if a user is root/admin differs between platforms